### PR TITLE
Hyper parameters of optimizer can be set from UI

### PR DESCRIPTION
**Pull request by Lewuathe**
- learning rate
- momentum
- batch size

They are configurable from UI.

screen shot 2017-09-04 at 21 33 47

---
This change is [Reviewable](https://reviewable.io/reviews/pair-code/deeplearnjs/90)

### CODE REVIEW

**Comment by nsthorat** [comment]
Reviewed 1 of 2 files at r1.
Review status: 1 of 2 files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

---

*[demos/model-builder/model-builder.html, line 243 at r1](https://reviewable.io:443/reviews/pair-code/deeplearnjs/90#-KtIpdlIuG10znfKj-3J:-KtIpdlJZ0ngy-iqgG2-:blt0x5o) ([raw file](https://github.com/pair-code/deeplearnjs/blob/8554dafc5e47b89adf0703e8ef07ed1d27f09de2/demos/model-builder/model-builder.html#L243)):*
> ```HTML
>       </paper-dropdown-menu>
> 
>       <div class="subtitle">Model Parameter</div>
> ```

Can you just make this `Hyperparameters`

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/pair-code/deeplearnjs/90)*

**Comment by nsthorat** [comment]
Review status: 0 of 2 files reviewed at latest revision, 2 unresolved discussions.

---

*[demos/model-builder/model-builder.html, line 243 at r2](https://reviewable.io:443/reviews/pair-code/deeplearnjs/90#-KtMU-161DrDG5Af1TjZ:-KtMU-161DrDG5Af1Tj_:b4fphd8) ([raw file](https://github.com/pair-code/deeplearnjs/blob/fbf4ff0af29c4a03b03bc3d650eb2e5d384fe9be/demos/model-builder/model-builder.html#L243)):*
> ```HTML
>       </paper-dropdown-menu>
> 
>       <div class="subtitle">HyperParameters</div>
> ```

sorry to be pedantic, can you lower case the p :)

https://en.wikipedia.org/wiki/Hyperparameter

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/pair-code/deeplearnjs/90)*

**Comment by Lewuathe** [comment]
Review status: 0 of 2 files reviewed at latest revision, 2 unresolved discussions, some commit checks failed.

---

*[demos/model-builder/model-builder.html, line 243 at r2](https://reviewable.io:443/reviews/pair-code/deeplearnjs/90#-KtMU-161DrDG5Af1TjZ:-KtMXXpzJJQMUUoZUSVK:b-ltmp4g) ([raw file](https://github.com/pair-code/deeplearnjs/blob/fbf4ff0af29c4a03b03bc3d650eb2e5d384fe9be/demos/model-builder/model-builder.html#L243)):*
<details><summary><i>Previously, nsthorat (Nikhil Thorat) wrote…</i></summary><blockquote>

sorry to be pedantic, can you lower case the p :)

https://en.wikipedia.org/wiki/Hyperparameter
</blockquote></details>

Ah sorry. I'll update..

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/pair-code/deeplearnjs/90)*

**Comment by nsthorat** [comment]
:lgtm_strong:

---

Reviewed 1 of 2 files at r1, 1 of 1 files at r3.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/pair-code/deeplearnjs/90#-:-KtMa8mBXeLXRpqJhSj7:b-jbsf8z)*
